### PR TITLE
fixes #17396 - remove the -d short option for dont-save-answers

### PR DESCRIPTION
**Pull request by bkearney**
Users are used to passing in -d to mean debug. The -d option maps
to dont-save-answers which has led to installer confusion. This
patch removes the short form of the argument and forces the user
to use the long form.

### CODE REVIEW

**Comment by stbenjam** [comment]
[test]

File: README.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ noop so no change will be done to your system. The default value here is set to
 false!
 
 Sometimes you may want kafo not to store answers from the current run. You can
-disable saving answer by passing a ```--dont-save-answers``` argument (or -d for short).
+disable saving answer by passing a ```--dont-save-answers``` argument.
```
Comment (stbenjam) [inline README.md]: While you're in here, could you make "answer" plural?

**Comment by bkearney** [comment]
@stbenjam I pushed a new change with the README file updated to have answers plural. No other changes.

**Comment by mmoll** [comment]
I guess this needs a (major?) version bump for the removal after merging.

**Comment by stbenjam** [comment]
Yea, I was wondering about that - 0.x isn't [strictly held to semver](http://semver.org/#spec-item-4), but this could cause someone issues if they're relying on `-d` for something.

**Comment by stbenjam** [comment]
Ah kafo's 1.0.5, so I guess this would require a bump to 2.0.

**Comment by stbenjam** [comment]
If we wanted to avoid the major bump for this, we could leave `-d` if we output a warning to the user that answers won't be saved when the option is used.

**Comment by bkearney** [comment]
@stbenjam let me know what you would like. I assume if the user does not RTFM then a warning is not gonna help.

**Comment by ares** [comment]
I think it's fine to bump to 2.0, let me deal with other PRs first, release another 1.x and then get back to this.

**Comment by stbenjam** [comment]
@ares Thanks!

**Comment by stbenjam** [comment]
@ares Can we get this in yet?

**Comment by bkearney** [comment]
@ares any chance to get this in before 1.15?

**Comment by stbenjam** [comment]
:+1: Let's get this in for 1.15.

 @bkearney can you also edit lib/kafo/version.rb and bump this to 2.0?

**Comment by bkearney** [comment]
@stbenjam same PR?

**Comment by stbenjam** [comment]
Yea, I meant this PR, or whoever releases it can do it separately.

**Comment by ares** [comment]
Don't worry, I'll do it on release. I'm sorry it took me so long. Could you please also update the [README](https://github.com/theforeman/kafo#testing-aka-noop-etc), it mentions the `-d`

File: README.md [hunk 2]
```diff
@@ -194,7 +194,7 @@ noop so no change will be done to your system. The default value here is set to
 false!
 
 Sometimes you may want kafo not to store answers from the current run. You can
-disable saving answer by passing a ```--dont-save-answers``` argument (or -d for short).
+disable saving answers by passing a ```--dont-save-answers``` argument.
```
Comment (bkearney) [inline README.md]: @ares do you want another change besides this?

**Comment by bkearney** [comment]
@stbenjam per the comment from will do nothing. @ares see comment on README.md file please.

File: README.md (hunk 2 above)
Comment (ares) [inline README.md]: Nope, that's it. Thank you

**Comment by ares** [comment]
Thanks @bkearney, merging.

**Pull request merged**